HHS/Head-Start-TTADP
# Add digest emails plus email logging

## Description of change
This PR creates cron jobs that run daily, weekly, and monthly. The jobs create email digests that are sent to users based on the type and frequency. The digests are compiled and put on a queue to be sent. Once sent, mailer log records are created in a separate table. 

This still needs to be merged with Jon's PR so that we can read the preferences. Right now the part where users are retrieved based on their preferences is hard-coded.

* Note - there are still tests that need to be added and to figure out why one test is failing in CI


## How to test
If testing locally:
- make sure there are reports that would qualify to be included in a daily collaborator digest
- modify the `dailySched `variable in `app.js` to run every 10 or 30 seconds, e.g. `'*/10 * * * * *'`
- yarn server
- yarn worker
- wait until a digest email arrives
- verify that the `MailerLog `table was updated 


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-965
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-979


## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] Boundary diagram updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
